tests/models/nextflow/conftest.py

```diff
@@ -1,39 +1,31 @@
-from typing import Dict, Optional
+from pathlib import Path
 
 import pytest
 
 
-@pytest.fixture(name="nextflow_deliverables")
-def nextflow_deliverables() -> Dict[str, str]:
-    return {
-        "format": "tsv",
-        "id": "CASEID",
-        "path": "PATHTOCASE/fusioncatcher/CASEID.fusioncatcher.fusion-genes.txt",
-        "path_index": "~",
-        "step": "fusioncatcher",
-        "tag": "fusioncatcher",
-    }
-
-
-@pytest.fixture(name="nextflow_deliverables_with_empty_entry")
-def nextflow_deliverables_with_empty_entry() -> Dict[str, Optional[str]]:
-    return {
-        "format": "",
-        "id": "CASEID",
-        "path": "PATHTOCASE/fusioncatcher/CASEID.fusioncatcher.fusion-genes.txt",
-        "path_index": "~",
-        "step": "fusioncatcher",
-        "tag": "fusioncatcher",
-    }
-
-
-@pytest.fixture(name="nextflow_deliverables_with_faulty_entry")
-def nextflow_deliverables_with_faulty_entry() -> Dict[str, str]:
-    return {
-        "faulty_header": "tsv",
-        "id": "CASEID",
-        "path": "PATHTOCASE/fusioncatcher/CASEID.fusioncatcher.fusion-genes.txt",
-        "path_index": "~",
-        "step": "fusioncatcher",
-        "tag": "fusioncatcher",
-    }
+@pytest.fixture(name="deliverable_id")
```

Comment:
We have decided against using `name` in the decorator.
```suggestion
@pytest.fixture
```